packages/server/routes/admin.js

```diff
@@ -1141,6 +1142,19 @@ const autoBackupForm = (req) => {
           auto_backup_destination: "S3",
         },
       },
+      {
+        type: "String",
+        label: "Backup password",
+        fieldview: "password",
+        name: "backup_password",
+        sublabel: req.__(
+          "Password to encrypt the backup file. Leave empty for no encryption"
+        ),
+        showIf: {
```

Comment:
no need for this showIf - backup password should always be a possibility even if auto backups not configured